code/modules/projectiles/guns/ballistic/bow.dm

```diff
@@ -0,0 +1,65 @@
+/obj/item/gun/ballistic/bow
+	name = "wooden bow"
+	desc = "some sort of primitive projectile weapon. used to fire arrows."
+	icon_state = "bow"
+	item_state = "bow"
+	w_class = WEIGHT_CLASS_BULKY
+	weapon_weight = WEAPON_HEAVY //need both hands to fire
+	force = 5
+	mag_type = /obj/item/ammo_box/magazine/internal/bow
+	fire_sound = 'sound/weapons/bowfire.ogg'
+	slot_flags = ITEM_SLOT_BACK
+	item_flags = NONE
+	casing_ejector = FALSE
+	internal_magazine = TRUE
+	pin = null
+	no_pin_required = TRUE
+	trigger_guard = TRIGGER_GUARD_NONE //so ashwalkers can use it
+
+/obj/item/gun/ballistic/bow/shoot_with_empty_chamber()
+	return
+
+/obj/item/gun/ballistic/bow/chamber_round()
+	chambered = magazine.get_round(1)
+
+/obj/item/gun/ballistic/bow/process_chamber()
+	chambered = null
+	magazine.get_round(0)
+	update_icon()
+
+/obj/item/gun/ballistic/bow/attack_self(mob/living/user)
+	if (chambered)
+		var/obj/item/ammo_casing/AC = magazine.get_round(0)
+		user.put_in_hands(AC)
+		chambered = null
+		to_chat(user, "<span class='notice'>You gently release the bowstring, removing the arrow.</span>")
+	else if (get_ammo())
```

Comment:
maybe a do_mob to pull back the string, which goes back to taught if not in an active hand

Reply:
working on this